DuckDuckGo/AddOrEditBookmarkViewController.swift

```diff
@@ -121,6 +121,7 @@ class AddOrEditBookmarkViewController: UIViewController {
         WidgetCenter.shared.reloadAllTimelines()
         self.delegate?.finishedEditing(self, entityID: viewModel.bookmark.objectID)
         dismiss(animated: true, completion: nil)
+        (UIApplication.shared.delegate as? AppDelegate)?.requestSyncIfEnabled()
```

Comment:
Ideally I'd like this to be passed down as dependency here and in other places, but it's not important for this PR - let's tweak it when preparing further tweaks. :)